cypress/integration/Case08_Receive_money.js

```diff
@@ -34,7 +36,11 @@ describe('Test case 8: Ability to send money request and reseive money', () => {
     ReceiveMoneyPage.nextButton.click()
     ReceiveMoneyPage.shareLinkButton.click()
     ReceiveMoneyPage.doneButton.should('be.visible')
-    cy.task('getClipboard').then(reseiveMoneyUrl => {
+
+    //get link from clipboard
+    //cy.task('getClipboard').then(reseiveMoneyUrl => {
+
+    cy.get('[data-gdtype="copy-link"]').invoke('attr', 'data-testid').then(reseiveMoneyUrl => {
```

Comment:
i've asked to use 'starts with' css selector for do not use additional data-attributes

According to this comment:
https://github.com/necolas/react-native-web/issues/1267#issuecomment-470251476
this is an workaround in React native web for now. And it could be changed. 
React native provides us testID prop should be used for mark UI controls for E2E tests
Let's use it and don't use data attributes if we could avoid them

In our case we could set link to the test id e.g. <Button testID="https://some-link" />
then find id via css selectors e.g. cy.get('[data-testid^="https:"]')
no additional data attributes is needed

Reply:
fixed